### DISCUSSION TITLE
Suggestions? ... ! ;)

### DISCUSSION THREAD

**Discussion by Flashhawk2K** [supermerill/SuperSlicer#2901] [Community contribution]
Hy there!
I don't know if I found the correct channel to post my thoughts, but if somebody could move it where it belongs i would be grateful!

So here are some things i am missing from SuperSlicer that I either experienced in other Slicers and found useful or that I would find useful to have but never seen anywhere:
1. Infill-Line multiplier.
Cura has this and especially with the option to leave off top and bottom layers it is a nice way to show off the infill without compromising the stability of box bottoms and stuff like that. Also aesthetically a nice tweakable option...
2. The option to change the coordinate origin of imported models. 
At the moment it is always in the middle of the model. And that means X, Y and Z! I would like to change the Origin for example to the left-top-most point that is on the build plate. (So the origin of cube would be one of its bottom corners) That would make arranging models that need to be joined via the "add" functionality much easier without adding catch-points like in real 3D Editor.
3.The "add" functionality should be like a group function
What i mean by that is, that I can not take two models that I have imported already and join them into one with the add function. Sometimes that would be very useful, especially when you have arranged and scaled a model already and would like to join it now to another model. At the moment your only option is to write down all relevant coordinates and dimensions, delete the model, add it via the add function to the other model, then re-scale and re-place it. This could be so much easier if you could just select the two models, and then right-click add them to one another or if you had an additional point in the add menu that said "select already imported model"

I hope that you find these as useful I would find them, so maybe one day I can see those added to functionality of a great slicer
Flashhawk2k

**Comment by neophyl** [comment]
You can select multiple objects and merge them already. Lots of stuff a available through the right click options.

**Comment by Flashhawk2K** [comment]
Yes, i found that literally the day after i posted... The other two suggestions would be still needed ;D
